### ISSUE TITLE
No instructions for second function

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/node-promises-diagnostic#101]
Diagnostic.js needs to instruct students to make the second function into a promise as well.

**Comment by jrhorn424** [comment]
Likely closable after #104

**Comment by gaand** [comment]
I disagree, I think my comment on #106 is the way to go, teaching the readline form.

**Comment by raq929** [comment]
If we teach the readline form, I would advocate for ditching the first function and just asking them to promisify the second one. It's seems clear that they don't have time to do both.
